tests/imageupload_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("[Serial]ImageUpload", func() {
 				"--image-path", imagePath,
 				"--size", pvcSize,
 				"--uploadproxy-url", fmt.Sprintf("https://127.0.0.1:%d", localUploadProxyPort),
-				"--wait-secs", "30",
+				"--wait-secs", "60",
```

Comment:
Regarding this:

> Sometimes the upload pod has a slow start (probably image pulling).

It is fine by me if the timeout gets increased. Just note that in general, increasing the timeout is only the start. Is this the first test which requires the cdi upload image? 

If yes, a proper fix may be to pre-pull this image to all nodes before starting the tests. In general everything except CDI is right now either already baked into kubevirtci. ,Only the kubevirt images themselves are pre-fetched to all kubevirtci nodes during `make cluster-sync` to avoid weird unpredictable timeouts due to varying pull speeds.

It would be great if we can follow up with a more targeted assessment and a more targeted fix. What do you think?

Reply:
In CDI we pre-pull all the CDI images during sync, I know I stole that code from KubeVirt, but I can't find it anymore for pre-pulling KubeVirt images.
https://github.com/kubevirt/containerized-data-importer/blob/main/cluster-sync/ephemeral_provider.sh#L18-L23

Reply:
I'll add the comment to https://github.com/kubevirt/kubevirt/issues/4868, and I'll try to read some logs.